packages/cli/generation/ir-generator/src/generateIntermediateRepresentation.ts

```diff
@@ -95,12 +96,15 @@ export async function generateIntermediateRepresentation({
                 if (errors == null) {
                     return;
                 }
+                const isStatusCodeDiscriminated =
```

Comment:
not a huge deal, but if we add a new strategy we won't get any compile breaks in this package. Alternative way is to pass in the `RawSchemas.ErrorDiscriminationConfigSchema` into convertErrorDeclaration, and have a helper function there (e.g. `convertDiscriminantValue`) that switches on the `strategy` and assertsNever in the default case.

Reply:
makes sense -- i didn't want to pass it in to the converter and do it outside since its going to be the same for every error. but maybe i can convert outside